deep_speech_2/test.py

```diff
@@ -104,8 +105,7 @@ def evaluate():
             language_model_path=args.lang_model_path,
             num_processes=args.num_proc_bsearch)
         target_transcripts = [
-            ''.join([data_generator.vocab_list[token] for token in transcript])
-            for _, transcript in infer_data
+            transcript for _, transcript in infer_data
```

Comment:
Line 107 - 109 could be put into single line?

Reply:
Done.